thirdeye/thirdeye-frontend/app/pods/components/modals/entity-mapping-modal/component.js

```diff
@@ -37,7 +37,7 @@ import _ from 'lodash';
 const MAPPING_TYPES = [
   'METRIC',
   'DIMENSION',
-  'SERVICE',
+  // 'SERVICE', // TODO: Enable once we want to fix services endpoint
```

Comment:
Not sure i understand this comment. Is the service not working? We should just fix it.

Reply:
Ah, yes. Sorry for lack of context. This is related to a query that was failing in the backend and we no longer need: `.../services/all`. After conversations, we don't want to display this in the entity mapping modal dropdown any longer. I can safely just remove it.